content/en/blog/2021/simple-vms-part-2/index.md

```diff
@@ -0,0 +1,290 @@
+            value: "sni-dnat"
+        service:
+          type: LoadBalancer
+          ports:
+            # health check port (required to be first for aws elbs)
+            - name: status-port
+              port: 15021
+              targetPort: 15021
+            # Port for mTLS passthrough
+            - port: 15443
+              targetPort: 15443
+              name: tls
+{{< /text >}}
+
+We also need to tell the east/west gateway to forward secure requests to the intended destination. To do this we create a simple Istio Gateway configuration.
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: eastwestgateway
+  namespace: istio-system
+spec:
+  selector:
+    istio: eastwestgateway
+  servers:
+  - port:
+      number: 15443
+      name: tls
+      protocol: TLS
+    tls:
+      mode: AUTO_PASSTHROUGH
+    hosts:
+    - "*"
+{{< /text >}}
+
+The `AUTO_PASSTRHOUGH` config automatically registers the mesh applications with the east/west gateway. You can view the available services by running the following `istioctl` command.
+
+{{< text bash >}}
+$ istioctl proxy-config listener -n istio-system <east_west_gateway_pod>
+{{< /text >}}
+
+Its important to note the SNI values for the mesh applications you wish to communicate with as they will be needed in the envoy configuration.
+
+{{< text bash >}}
+$ istioctl proxy-config listener -n istio-system istio-eastwestgateway-868fc64fc4-fn6q4 --port 15443
+ADDRESS PORT  MATCH
+0.0.0.0 15443 SNI: outbound_.9402_._.cert-manager.cert-manager.svc.cluster.local;
+0.0.0.0 15443 SNI: outbound_.9153_._.kube-dns.kube-system.svc.cluster.local;
+0.0.0.0 15443 SNI: outbound_.80_._.frontend.simple-app.svc.cluster.local;
+0.0.0.0 15443 SNI: outbound_.53_._.kube-dns.kube-system.svc.cluster.local;
+0.0.0.0 15443 SNI: outbound_.50051_._.fortune-teller.fortune-teller.svc.cluster.local;
+0.0.0.0 15443 SNI: outbound_.443_._.metrics-server.kube-system.svc.cluster.local;
+0.0.0.0 15443 SNI: outbound_.443_._.kubernetes.default.svc.cluster.local;
+0.0.0.0 15443 SNI: outbound_.443_._.istiod.istio-system.svc.cluster.local;
+0.0.0.0 15443 SNI: outbound_.443_._.cert-manager-webhook.cert-manager.svc.cluster.local;
+0.0.0.0 15443 SNI: outbound_.15443_._.istio-eastwestgateway.istio-system.svc.cluster.local;
+0.0.0.0 15443 SNI: outbound_.15021_._.istio-eastwestgateway.istio-system.svc.cluster.local;
+0.0.0.0 15443 SNI: outbound_.15014_._.istiod.istio-system.svc.cluster.local;
+0.0.0.0 15443 SNI: outbound_.15012_._.istiod.istio-system.svc.cluster.local;
+0.0.0.0 15443 SNI: outbound_.15010_._.istiod.istio-system.svc.cluster.local;
+{{< /text >}}
+
+## Virtual Machine Configuration
+
+![Virtual Machine Configuration](./vm-configuration.png)
+
+Because we will not be connecting envoy to an external xDS server, we will need to provide the configuration manually. We will need to create an inbound listener for the virtual machine application to communicate through.
```

Comment:
```suggestion
Because we will not be connecting the envoy in the virtual machine to an external xDS server, we will need to provide its configuration manually. We will need to create an inbound listener for the virtual machine application to communicate through.
```